roottest/root/tree/chain/execmissingIndex.ref

```diff
@@ -1,13 +1,13 @@
 
 Processing execmissingIndex.C...
 Scanning using the original friend
-Error in <TChainIndex::TChainIndex>: The indices in files of this chain aren't sorted.
-Error in <TChainIndex::TChainIndex>: The indices in files of this chain aren't sorted.
-Error in <TChainIndex::TChainIndex>: The indices in files of this chain aren't sorted.
-Error in <TChainIndex::TChainIndex>: The indices in files of this chain aren't sorted.
-Error in <TChainIndex::TChainIndex>: The indices in files of this chain aren't sorted.
-Error in <TChainIndex::TChainIndex>: The indices in files of this chain aren't sorted.
-Error in <TTreePlayer::BuildIndex>: Creating a TChainIndex unsuccessful - switching to TTreeIndex
+Warning in <TChainIndex::TChainIndex>: The indices in files of this chain aren't sorted.
```

Comment:
Note that it is a significant performance degradation.  If the performance degradation is okay with the user (i.e. a case when sorting the files and/or entries is actually more expensive) then it does make sense to not print the error.  So rather than a verbose flag, it should be more a 'please allow unsorted files' where the error is then expected and thus not printed.

Reply:
I've reworded the flag and the documentation.